src/commands/slowlog-get.json

```diff
@@ -15,10 +15,13 @@
         ],
         "command_flags": [
             "ADMIN",
-            "RANDOM",
             "LOADING",
             "STALE"
         ],
+        "command_tips": [
+            "REQUEST_POLICY:ALL_MASTERS",
```

Comment:
Unclear why slowlog commands aren't ALL_NODES as well, replicas can have slow commands.

Reply:
presumably, you only care about latency in master shards. but maybe after failover you don't wanna lose visibility.
as i noted i also wanna apply similar change to SCRIPT LOAD and SCRIPT FLUSH